statscache/templates/reference.html

```diff
@@ -0,0 +1,146 @@
+{% extends "layout.html" %}
+{% block title %}reference{% endblock %}
+{% block head %}
+    <style type="text/css">
+    li {
+        padding-bottom: 6pt;
+    }
+    li:last-child {
+        padding-bottom: 0;
+    }
+    </style>
+    <meta name="description" content="API Reference">
+{% endblock %}
+{% block body %}
+    <section class="lead">
+        <h1>overview</h1>
+        <p>
+            To allow integration with other web applications and services,
+            statscache offers a simple, easy-to-use REST API with support for
+            multiple convenient data formats.
+        </p>
+    </section>
+{# This page really ought to be more thorough #}
```

Comment:
Sounds good :) part of me really wanted to keep at it, but I'd already spent days on just that page (not much of a writer)